setup.py

```diff
@@ -0,0 +1,16 @@
+from distutils.core import setup
+from setuptools import find_packages
+
+setup(
+    name='clipper_admin',
+    version='0.1',
+    description='A low latency online prediction serving system',
```

Comment:
Make the description "Admin commands for the Clipper prediction-serving system."